code-challenges.test.js

```diff
@@ -35,9 +72,41 @@ const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+describe("oddElevation", () => {
+
+    it(" oddElevation will take arrays fullArr2, oddElevation will return a new array containing only odd numbers sorted least to greatest", () => {
+
+        expect(oddElevation()).toEqual([-9, 7, 9, 199])
+
+    })
+})
+
 
 // b) Create the function that makes the test pass.
 
+//---- I created a function called oddElevation 
+//---- oddElevation will take arrays fullArr1 and fullArr2...
+//---- I created a function jusNumberz to sort numbers from least to greatest
+//---- I created jusOddNumberz to .filter and return only odd numbers
+//---- I created jusOddNumbersSorted to sort least to greatest
+//---- oddElevation will return a new array containing only odd numbers sorted least to greatest
+
+ const oddElevation = () => {
+
+     function jusNumberz(fullArr2) {
+         return fullArr2.filter((element) => typeof element === "number")
+     }
+     let jusOddNumberz = jusNumberz(fullArr2).filter(value => value % 2 || -2 === 1)
+     let jusOddNumberzSorted = jusOddNumberz.sort((a, b) => a - b)
+
+     return jusOddNumberzSorted
+ }   
+ console.log(oddElevation())
+ 
+ //---- fullArr1 console.logged: [-9, 7, 9, 199]
+ //---- fullArr2 console.logged: [ -823, 7, 23 ]
+
+
```

Comment:
great work! Your logic is awesome and works! One issue is the OR statement on line 99.  I recommend you look up how to correctly write that code.  You just need to make ithe function dynamic so any array can pass.  If you put a param in the () in line 94 and then repeat that param instead of using fullArr2, you'll have a dynamic function!  
You're right there - just make that adjustment and you're golden!